_static/api-examples.md

```diff
@@ -126,7 +126,7 @@ You must authenticate with the CDX CROMERR service for signing a manifest. Value
 ```bash
 curl -i -X POST -H 'Content-Type: application/json' \
   --data @auth-creds.json \
-  https://e-manifest.18f.gov/api/0.1/user/authenticate
+  https://e-manifest.18f.gov/api/v1/users/authentications
```

Comment:
Since the important thing about authenticating isn't the 'session' per se, but the actual response to be used for creating a signature, how about something like:

```
POST .../token
```

?